### PR TITLE
feat(bnc-onboard): add exodus wallet support

### PR DESCRIPTION
**Pull request by mzndako**
### Description
Some dapps are still using onboard V1 (bnc-onboard) and are not ready to migrate to V2 yet. This PR adds Exodus wallet support on onboard V1 to make Exodus wallet available for those dapps.

image

## To Test
1. Install [Exodus Browser Extension](https://www.exodus.com/browser-extension/)
2. Connect to a Wallet
3. Select Exdous
4. Connect successfully

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.38.4",
+  "version": "1.38.4-0.0.1",
```
Comment (Adamj1232) [inline package.json:3]:
```suggestion
  "version": "1.39.0",
```
We can push a minor version bump for this change, no need to add the staging version at the end

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Looks good one small change to the version as I wasnt able to push to your fork.

File: package.json (hunk 1 above)
Comment (mzndako) [reply package.json]: Changes implemented.

**Review by Adamj1232 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**